tasty-query/shared/src/main/scala/tastyquery/reader/pickles/PickleReader.scala

```diff
@@ -384,6 +384,7 @@ private[pickles] class PickleReader {
         val pre = readPrefix()
         val designator = readMaybeExternalSymbolRef()
         designator match
+          case sym: PackageSymbol          => sym.packageRef
```

Comment:
does this fix the test case about erasure of `java.lang.reflect.Field`?

Reply:
oh no wait that was in tasty signatures, but I guess this should cover scala 2 also

Reply:
I can add regression test for that in a new PR